core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -485,6 +496,13 @@ class SpendReportingService(
           .toMap
       }
 
+      _ = if (projectNames.isEmpty) {
+        throw RawlsExceptionWithErrorReport(
+          StatusCodes.NotFound,
+          s"no spend data found for billing project ${project.value} between dates ${toISODateString(start)} and ${toISODateString(end)}"
+        )
+      }
+
```

Comment:
this prevents us from asking BigQuery for results in cases where there would be no results